Drop support for Django 1.8/1.9/1.10

vThaian opened the pull request.

czpython commented:
@vThaian 
There's a nasty bug on 3.6.0 which needs to be released **before** releasing 3.7 (breaking changes)

vThaian commented:
@czpython you write about:
* Updated setup.py to use html5lib>=0.999999999
* Fixed ValueError on  static file resolution at import time
right? Ok, will make release for 3.6.1 first to resolve that

czpython commented:
Yup

vxsx commented on djangocms_text_ckeditor/tests/frontend/integration/smoke.js:
```diff
@@ -171,11 +173,12 @@ casper.test.begin('CKEditor loads correctly in collapsed fieldset', function (te
         .waitForSelector('.success', function () {
-            test.assertSelectorHasText('.success', 'The pizza "Pizza object" was added successfully.', 'Pizza added');
+            test.assertSelectorHasText('.success', 'was added successfully.', 'Pizza added');
+            test.assertSelectorHasText('.success', 'Pizza object (2)', 'Pizza added');
```
```suggestion
            test.assertSelectorHasText('.success', 'Pizza object', 'Pizza added');
```
that way it would be consistent across django versions (<2.0 didn't have the number in the name)

vxsx commented on djangocms_text_ckeditor/tests/frontend/integration/smoke.js:
```diff
@@ -171,11 +173,12 @@ casper.test.begin('CKEditor loads correctly in collapsed fieldset', function (te
         .thenOpen(globals.baseUrl + 'admin/test_app/pizza/')
         .then(function () {
-            this.clickLabel('Pizza object');
+            this.clickLabel('Pizza object (2)');
```
this needs to be clickable regardless of it having a number in the name or not, which is probably better to do with a combination of http://docs.casperjs.org/en/latest/modules/casper.html#click and an xPath selector, e.g.

```suggestion
this.click(xPath('//[contains(text(), 'Pizza object')]'))
```

where `xPath` is 
```
var casperjs = require('casper');
var xPath = casperjs.selectXPath;
```
